docs/apps/components/color.md

```diff
@@ -0,0 +1,190 @@
+7. **Reducing Cognitive Load**: Limiting color palettes can alleviate cognitive load on users, particularly for long or complex forms, facilitating better focus on the form itself.
+
+### Interactivity
+
+**Interactivity** is the ability of a web page or application to respond to user input.
+
+There are a number of ways to add interactivity to the **Color** component.
+
+Some of the most common methods include:
+
+1. **Disable/enable** the color picker based on other form values using the enable() and disable() methods. For example, disable if a checkbox is unchecked.
+
+2. **Show/hide** the picker conditionally by calling show() and hide() methods. For example, only show it if a radio button is selected.
+
+3. **Validate** the selected color using validate() and display errors for invalid values.
+
+4. **Change** available colors in the palette dynamically based on other form selections.
+
+5. **Preselect** the color based on logic and calculations when the form initializes.
+
+6. **Update** styles of other elements based on the selected color using CSS custom properties.
+
+7. **Display** color preview swatches elsewhere in the form when the color changes.
+
+## API
+
+### Events
+
+| **Name**| **Description**|
+|---------|----------------|
+| `onChange (element)`| The onChange event occurs when the value of a field is changed|
+
+### Methods
+
+| **Name**| **Description**|**Parameters**|
+|---------|----------------|--------------|
+|`this.addClassName`|Add style class to a form item|`(fields: String, String[]; className: String)`|
+|`this.removeClassName`|Remove style class from a form item|`(fields: String, String[]; className: String)`|
+|`this.getComponent`|Returns a component whose id has been passed as a parameter|`(component_ID: String(), String[]; Object: Object)`|
+|`this.getValues`|Get the values of all fields when values change|`Object`|
+|`this.hide`|Hides the field|`(fields: String()`|
+|`this.show`|Displays the field|`(fields: String()`|
+| `this.disable`| Disable input field from user interaction|`(fields: String(), String[])`|
+|`this.getValue`|Get A Value From a component|`(fieldName String()`|
+|`this.setData`|Set the default color|`(Value: Hexadecimal color code)`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+
+#### Steps to place/use the methods for components of the Page Builder
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
```

Comment:
Remove the word **Setting** from this line

Reply:
not required: confirmed with Cacious over call.